networks/meld/tokens/0xF463A50a2C51453cF6004f330aF34FB895Fc65aF.json

```diff
@@ -0,0 +1,16 @@
+{
+    "CMCslug": "MLENS",
```

Comment:
This slug does not exist on CoinMarketCap.

Reply:
We just added liquidity, may be that is the reason? it should be available now?

Reply:
> We just added liquidity, may be that is the reason? it should be available now?

CMCSlug is for token, already listed on CmC. CmC does not even support the MELD chain yet, so there is no way the slug is available. Also, most of the Cardano tokens are not even listed after being active projects for years. So this should be empty and price will not be tracked for now.

Reply:
> We just added liquidity, may be that is the reason? it should be available now?

Just FYI, this is a token slug:
![image](https://github.com/user-attachments/assets/28f0b8f3-bde8-402a-9833-5ff03996e139)